djadmin2/tests/test_types.py

```diff
@@ -40,7 +40,7 @@ def test_attributes(self):
             self.immutable_admin.d
 
 
-class Thing(models.Model):
+class BigThing(models.Model):
```

Comment:
AFAR I believe there was clash of names and this was raising some errors for django.